### PR TITLE
Fix more GoTime 13 unintelligible

### PR DESCRIPTION
**Pull request by amaltson**
This is about the most I can do with GoTime 13, I was hoping to fix all the `unintelligible` but some of them are too hard to hear or understand what was said, especially since I'm not sure versed in `unsafe` concepts in Go.

### CODE REVIEW

File: gotime/go-time-13.md [hunk 1]
```diff
@@ -312,7 +312,7 @@ There is also another idea that I think came from Katrina. Katrina works at Gith
 
 **Carlisia Pinto:** Were you at the San Francisco meetup yesterday, Francesc?
 
-**Francesc Campoy:** Yes, I was there, but I had to leave, so I missed \[unintelligible 00:42:04.16\] talk. I know he was recorded, so I'm looking forward to watching it, too.
+**Francesc Campoy:** Yes, I was there, but I had to leave, so I missed Burcu's talk. I know it was recorded, so I'm looking forward to watching it, too.
```
Comment (amaltson) [inline gotime/go-time-13.md]: This one was fun, I tracked down the GoSF meetup that they were talking about and found the [Peripheral I/O for Go by Jaana Burcu Dogan](https://www.meetup.com/golangsf/events/mbnfslyvkbbc/) talk. I wonder if it's worth expanding the name via author comment like `Burcu's [Jaana Burcu Dogan]` talk. Let me know.

Also, maybe worth a link in the show notes? I have the [video](https://www.youtube.com/watch?v=xteEImopTic) just got to find the time point where her talk is.

Comment (jerodsanto) [reply gotime/go-time-13.md]: Going the extra mile, love it! Yeah it'd be totally cool to add that extra info like you suggested and even add a link to the video right here in the transcript.

If you'd like to add it in the show notes, they're also open source :)

https://github.com/thechangelog/show-notes

Comment (amaltson) [reply gotime/go-time-13.md]: Thanks @jerodsanto! Will do, I had a link to a blog post earlier too, so I'll add that as well.

Comment (amaltson) [reply gotime/go-time-13.md]: Btw, it turned out @jerodsanto already had all the links in the [show notes](https://github.com/thechangelog/show-notes/blob/master/gotime/go-time-13.md)! Doh!

**Pull request merged**